### PR TITLE
[15.0][IMP] stock_inventory_discrepancy: Filter overdiscrepancy quants.

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Useful when performing inventory adjustments:

image

@ForgeFlow

### CODE REVIEW

File: stock_inventory_discrepancy/models/stock_quant.py [hunk 1]
```diff
@@ -23,9 +24,24 @@ class StockQuant(models.Model):
         compute="_compute_discrepancy_threshold",
     )
     has_over_discrepancy = fields.Boolean(
-        compute="_compute_has_over_discrepancy",
+        compute="_compute_has_over_discrepancy", search="_search_has_over_discrepancy"
     )
 
+    def _search_has_over_discrepancy(self, operator, value):
+        if operator not in ["=", "!="]:
```
Comment (LoisRForgeFlow) [inline stock_inventory_discrepancy/models/stock_quant.py:31]: if you support "!=" you should consider de operator in the above computation

Comment (AaronHForgeFlow) [reply stock_inventory_discrepancy/models/stock_quant.py:31]: that's true, thanks

**Review by DavidJForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
Probably not the most efficient search, but let's merge and refine later if needed.

**Comment by LoisRForgeFlow** [comment]
@AaronHForgeFlow can you fwport to 16 and 17?

**Comment by AaronHForgeFlow** [comment]
> @AaronHForgeFlow can you fwport to 16 and 17?

Yes!

**Pull request merged**